plugin/parser/differ/mysql/differ.go

```diff
@@ -215,31 +239,9 @@ func (*SchemaDiffer) SchemaDiff(oldStmt, newStmt string) (string, error) {
 		default:
 		}
 	}
-
 	return deparse(newNodeList, inplaceUpdate, inplaceAddNodeList, inplaceDropNodeList, dropNodeList, format.DefaultRestoreFlags|format.RestoreStringWithoutCharset)
 }
 
-// We're not supporting all the MySQL statements for now.
-// validateStmtNodes validates the stmt nodes, return nil if they are valid.
-func validateStmtNodes(nodes []ast.StmtNode) error {
```

Comment:
Why do you remove this function

Reply:
As discussed offline, we just support `sync schema` but not `SDL for gitops` now. We should just align with `mysqldump` so there is no need to validate user input. I will add it back when we prepare to support `SDL for gitops`.